### ISSUE TITLE
Bitarray does not support sys.getsizeof()

### ISSUE THREAD

**Issue by cycloid** [ilanschnell/bitarray#100]
It looks like bitarray() does not accurately support the sys.getsizeof() calculation - the size returned appears to be the size of the base object, regardless of the length of the bitarray vector itself.

In [1]: import sys
In [2]: from bitarray import bitarray
In [3]: sys.getsizeof(bitarray(10))
Out[3]: 64
In [4]: sys.getsizeof(bitarray(10 * 1000 * 1000))
Out[4]: 64

Compare with list():
In [5]: sys.getsizeof(list(range(10)))
Out[5]: 136
In [6]: sys.getsizeof(list(range(10 * 1000 * 1000)))
Out[6]: 80000056

[Python 3.8.5; bitarray 1.5.2, via conda]

**Comment by ilanschnell** [comment]
Thanks for reporting this issue.  I think that all that needs to be done is implement a `__sizeof__` method in order to support `sys.getsizeof()`.

**Comment by cycloid** [comment]
Thanks very much!  I know it's not a critical feature bit I always surprise myself by how often I look at sizes, etc when comparing options.

Works for me on macOS 10.15.6

**Comment by ilanschnell** [comment]
You're welcome, and thanks for letting me know that it works for you.  I've added tests, and tried it on Linux, Mac and Windows.  I will make a 1.5.3 release soon (probably next week) which includes this.

**Comment by ilanschnell** [comment]
@Jianningli I've added tests for this new feature, and all seems to work for different Python versions and all OSes.  For example (on OSX using Python 3.8) I have:
```
>>> import sys
>>> from bitarray import bitarray
>>> a = bitarray(1000)
>>> sys.getsizeof(a)
189
```
Are you using the git master?  Which version of Python / operating system are you using?  Thanks

**Comment by Jianningli** [comment]
> @Jianningli I've added tests for this new feature, and all seems to work for different Python versions and all OSes. For example (on OSX using Python 3.8) I have:
> 
> ```
> >>> import sys
> >>> from bitarray import bitarray
> >>> a = bitarray(1000)
> >>> sys.getsizeof(a)
> 189
> ```
> 
> Are you using the git master? Which version of Python / operating system are you using? Thanks

Hi, thanks for replying. I installed using pip (https://pypi.org/project/bitarray/), on win10, python 3.6.
![screenshot](https://user-images.githubusercontent.com/58332790/91017294-58063380-e5ee-11ea-9b1b-a0e418977d8e.PNG)
Maybe I should try install from source later. thanks!

**Comment by ilanschnell** [comment]
@Jianningli Yes, the new feature is currently only on master.  It has not yet been part of a release (I only publish releases to PyPI).  I'll release bitarray 1.5.3 later this week which will include this feature.

**Comment by Jianningli** [comment]
> @Jianningli Yes, the new feature is currently only on master. It has not yet been part of a release (I only publish releases to PyPI). I'll release bitarray 1.5.3 later this week which will include this feature.

I see. Thanks for replying.

**Comment by ilanschnell** [comment]
I just released 1.5.3 with the new feature.
